src/System.Management.Automation/utils/VTUtils.cs

```diff
@@ -1,13 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System;
 using System.Collections.Generic;
 
 namespace System.Management.Automation
 {
     /// <summary>
     /// Class to help with VT escape sequences.
     /// </summary>
+    [Obsolete("This class is deprecated. Use 'System.Management.Automation.PSStyle' instead.", error: true)]
     public sealed class VTUtility
```

Comment:
I think it is better to remove the class at all. `Obsolete` doesn't work for scripts and I guess nobody use this in C#.

Reply:
I'm fine doing that, given it's not used anywhere on GitHub other than in PowerShell: https://grep.app/search?q=VTUtility%5D%3A%3AGetEscapeSequence.
But I'm also fine first making it obsolete and remove it in the next 7.x release.

Reply:
It seems there are other obsolete classes - is it time to review them too for the next release?

Reply:
I left the committee a message about this. Hopefully they will review all the obsolete types and methods.